runtime-light/stdlib/rpc/rpc-api.h

```diff
@@ -129,7 +138,17 @@ inline task_t<array<int64_t>> f$rpc_tl_query(const class_instance<C$RpcConnectio
 
 // === Rpc Misc ===================================================================================
 
-void f$rpc_clean() noexcept;
+inline void f$rpc_clean() noexcept {
+  RpcInstanceState::get().rpc_buffer.clean();
+}
+
+inline int64_t f$rpc_tl_pending_queries_count() noexcept {
+  return RpcInstanceState::get().response_waiter_forks.size();
+}
+
+inline constexpr bool f$set_fail_rpc_on_int32_overflow(bool /*unused*/) noexcept {
+  return true;
+}
```

Comment:
Why is it noop?
For now it leads to additional check in tl layer with tl error or warning

Reply:
You're right, fixed